src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -16,6 +16,10 @@
 /**
  * TranslatorBagInterface.
  *
+ * @method MessageCatalogueInterface[] getCatalogues() Returns all catalogues of the instance
+ * @method string[]                    getDomains()    Returns all unique domains of the instance
+ * @method array                       all()           Returns all messages of the instance
```

Comment:
I don't understand why we added these methods.

Reply:
These methods are used in Providers write/read methods, as they accept or use TranslationBagInterface. We discussed about it with @nicolas-grekas, may be he could re-explain more clearly than me why we added them?